manifests/P/PlasmoApp/plasmo_voice/1.x/1.2.x/1.2.9.yaml

```diff
@@ -0,0 +1,100 @@
+# yaml-language-server: $schema=../../../../../../doc/validation-schemas/4.0/version-manifest.json
+
+#1.18
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.18.1"
+    - "1.18"
+  environment:
+    server: optional
+    client: optional
+  channel: release
+  depends: ~
+  bundles: ~
+  breaks: ~
+  conflicts: ~
+  recommends: ~
```

Comment:
The `fabric.mod.json` has `"soundphysics": "*"` under `suggests`, and after some thought I'd say they meant `recommends`, so you should add it to our `recommends`:
```yaml
  recommends:
    - packageId: vlad2305m.soundphysics
      version: "*"
```